doc/man7/timew-dates.7.in

```diff
@@ -55,7 +55,7 @@ In addition to the standard date formats, the following are supported:
   <month-of-year>                       Previous 1st of the  month at 0:00:00
   hh:mm[:ss][am|a|pm|p]                 Short time format
   Nst, Nnd, Nrd, Nth                    Previous 1st, 2nd, 3rd ...
-  <epoch>                               POSIX time
+  <epoch>                               POSIX time (at least 315532800)
```

Comment:
I wonder whether we should add the corresponding date as well, e.g. something like
```
POSIX time (at least 315532800, i.e. after 1980-01-01)
```
The motivation for this limit seems to have `<epoch>` distinguishable from a date like `YYYYMMDD`. This may also be worth documenting.

Reply:
That's one digit too much though, but I had the same thought initially. Should I add information about the corresponding 1980-01-01?

Reply:
I guess they wanted a nice date instead of the "minimal required date". 🤷🏻‍♂️ 

The question is whether a short add-on like proposed above is enough or we need some more clarification (somewhere else). The code comment simply states `... so 12 is not converted to a date`.

Reply:
I added a bit more information to the man page to explain this choice.